### ISSUE TITLE
If previous commit has not comment yet, all history lost

### ISSUE THREAD

**Issue by danez** [netlify/delta-action#309]
**Describe the bug**

When a previous commit does not yet have the comment from delta-actions, the action will just write the current deltas but no history. So basically starting from scratch.

**Comment by eduardoboucas** [comment]
Yeah, that is annoying. The idea of keeping the history in the commit via the hidden comments was to avoid hammering the GitHub API each time to find the results of the previous commits, but perhaps that would be a better solution?
